metabase/metabase
# Convert `flex.module.css` to CSS modules

### Description

Converts `flex.module.css` to CSS modules - all of the selectors refer to flexbox layouts.

### How to verify

Because we use flexbox elements everywhere, this file touches just about every part of the app. Feel free to click around and make sure that things don't look weird.

Each commit refers to a new selector to hopefully make reviews a little easier.

### Checklist
